mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -96,11 +96,7 @@ impl Bridge<WakingMemoryStore> {
                     &system_address,
                     settings.keep_alive(),
                     settings.clean_session(),
-                    Credentials::Anonymous(format!(
-                        "{}/$edgeHub/$bridge/{}",
-                        device_id,
-                        settings.name()
-                    )),
+                    Credentials::Anonymous(format!("{}/$edgeHub/$bridge", device_id,)),
```

Comment:
Not sure if this is the correct fix.  Since we are going to use different MQTT client to connect to the local broker, we need to assign a unique client_id to each of them. What will happen right now for multiple bridges: all local clients will share the same client_id and start to fight for the session in the local broker. The next one will take over and previous connection will be closed. And same for the current one again and again.